packages/ui-default/templates/partials/footer.html

```diff
@@ -81,7 +81,7 @@ <h1>
           {%- for html in (model.system.get('ui-default.footer_extra_html')|default('')).split('\n') -%}
             <li class="footer__extra-link-item">{{ html|safe }}</li>
           {%- endfor -%}
-          <li class="footer__extra-link-item">Worker {{ process.env.NODE_APP_INSTANCE }} loaded in {{ Date.now() - handler.args.__start }}ms</li>
+          <li class="footer__extra-link-item">Worker {{ process.env.NODE_APP_INSTANCE }} has processed your request in {{ Date.now() - handler.args.__start }}ms</li>
```

Comment:
This might be too long for most desktop devices (which makes footer link displays into two lines). I still prefer using short forms (something like `Worker 0, 3ms`)